Reflections/R5 - May 14.md

```diff
@@ -0,0 +1,14 @@
+## 1. What have you accomplished this week? (Please list each accomplishment, and explain)
+I've had a much more productive week this week then last week. I've implemented my zone and renown system as outlined in the previous reflection document. I've also done some minor tweaking on obj_toolbar. I have also done some user tests to get some more directed feedback on my project from various people not in the class.
```

Comment:
Fantastic progress on this! Zones absolutely blew me away this past week. Great job implementing the core features you originally outlined.